### ISSUE TITLE
General accesability cleanup

### ISSUE THREAD

**Issue by kalaniJ94** [kalaniJ94/week1-refactor-seo#6]
According to client expectations, general accessibility should be improved.

**Comment by kalaniJ94** [comment]
Final changes added!
